inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -921,6 +928,7 @@ void convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function
         };
         const static std::vector<std::shared_ptr<Builder::INodeConverter>> convertors = {
                 std::make_shared<Builder::NodeConverter<::ngraph::op::v1::AvgPool>>(),
+                std::make_shared<Builder::NodeConverter<::ngraph::op::v1::Broadcast>>(),
```

Comment:
Please use visitor API instead of node converter.

Reply:
Thanks! Changed